gems/sorbet-runtime/lib/types/profile.rb

```diff
@@ -1,30 +1,25 @@
 # typed: true
 # frozen_string_literal: true
 
+# Deprecated, kept only for partial backwards compatibility
```

Comment:
When do you plan to remove this completely?

Reply:
yeah i don't think we every documented or publicized this much, so i think it's safe to just remove the ~5 usages of it in Stripe's codebase pre-emptively and then land this PR.